rules/windows/image_load/image_load_side_load_third_party_location.yml

```diff
@@ -0,0 +1,75 @@
+title: Third Party Software DLL Sideloading
+id: 72ca7c75-bf85-45cd-aca7-255d360e423c
+status: experimental
+description: Detects DLL sideloading of DLLs that are part of third party software such as antivirus
+references:
+    - https://hijacklibs.net/ # For list of DLLs that could be sideloaded (search for dlls mentioned here in there)
+author: Nasreddine Bencherchali, Wietze Beukema (project and research)
+date: 2022/08/17
+tags:
+    - attack.defense_evasion
+    - attack.persistence
+    - attack.privilege_escalation
+    - attack.t1574.001
+    - attack.t1574.002
+logsource:
+    category: image_load
+    product: windows
+detection:
+    # Bitdefender
+    selection_bitdefender:
+        ImageLoaded|endswith: '\log.dll'
+    filter_bitdefender:
+        ImageLoaded|startswith:
+            - 'C:\Program Files\Bitdefender Antivirus Free\'
+            - 'C:\Program Files (x86)\Bitdefender Antivirus Free\'
+    # F-Secure
+    selection_fsecure:
+        ImageLoaded|endswith: '\qrt.dll'
+    filter_fsecure:
+        ImageLoaded|startswith:
+            - 'C:\Program Files\F-Secure\Anti-Virus\'
+            - 'C:\Program Files (x86)\F-Secure\Anti-Virus\'
+    # Google Chrome
+    selection_chrome:
+        ImageLoaded|endswith: '\chrome_frame_helper.dll'
+    filter_chrome:
+        - ImageLoaded|contains: '\AppData\local\Google\Chrome\Application\'
+        - ImageLoaded|startswith:
+            - 'C:\Program Files\Google\Chrome\Application\'
+            - 'C:\Program Files (x86)\Google\Chrome\Application\'
+    # Lenovo
+    selection_lenovo:
+        ImageLoaded|endswith: '\commfunc.dll'
+    filter_lenovo:
+        - ImageLoaded|contains: '\AppData\local\Google\Chrome\Application\'
+        - ImageLoaded|startswith:
+            - 'C:\Program Files\Lenovo\Communications Utility\'
+            - 'C:\Program Files (x86)\Lenovo\Communications Utility\'
+    # McAfee
+    selection_mcafee:
+        ImageLoaded|endswith:
+            - '\ashldres.dll'
+            - '\lockdown.dll'
+            - '\vsodscpl.dll'
+    filter_mcafee:
+        ImageLoaded|startswith:
+            - 'C:\Program Files\McAfee\'
+            - 'C:\Program Files (x86)\McAfee\'
+    # Toshiba
+    selection_toshiba:
+        ImageLoaded|endswith: '\tosbtkbd.dll'
+    filter_toshiba:
+        ImageLoaded|startswith:
+            - 'C:\Program Files\Toshiba\Bluetooth Toshiba Stack\'
+            - 'C:\Program Files (x86)\Toshiba\Bluetooth Toshiba Stack\'
+    # Zoom (FP with System32)
+    #selection_zoom:
+    #    ImageLoaded|endswith: '\version.dll'
+    #filter_zoom:
+    #    ImageLoaded|startswith: 'C:\Users\'
+    #    ImageLoaded|contains: '\AppData\Roaming\Zoom\bin\'
+    condition: (selection_bitdefender and not filter_bitdefender) or (selection_fsecure and not filter_fsecure) or (selection_chrome and not filter_chrome) or (selection_lenovo and not filter_lenovo) or (selection_mcafee and not filter_mcafee) or (selection_toshiba and not filter_toshiba)
```

Comment:
I am not a big fan of so long condition.
Can you split ?
At least in three : 
- antivirus
- web brower
- 3 party tools

Reply:
Done.